### ISSUE TITLE
Difference between DeepPerception-FGVR and DeepPerception

### ISSUE THREAD

**Issue by StevenSmith2000** [thunlp/DeepPerception#1]
Hi, I found that there are two models in HuggingFace, i.e., DeepPerception-FGVR and DeepPerception. What is the difference? Is DeepPerception-FGVR also trained by KVG task? Thanks.

**Comment by MaxyLee** [comment]
Thank you for your interest in our work!

We trained the DeepPerception and DeepPerception-FGVR models using the KVG and FGVR datasets, respectively. To ensure optimal performance, please use the former for evaluating the KVG task and the latter for the FGVR task.

If you have any further questions, feel free to reach out to us — we’ll get back to you as soon as possible.

**Comment by StevenSmith2000** [comment]
Thanks for your reply.

I have some further questions:
(1) I wonder how to construct the training data for the FGVR task, is it organized as the multiple-choice task or open-set QA task? 
(2) Is FGVR CoT data constructed following the similar pipeline as KVG CoT data?
(3) How is the reward function designed for RL training stage of DeepPerception-FGVR?

Thanks a lot!

**Comment by MaxyLee** [comment]
Thank you again for your continued interest in our work!

1. We reformulated the FGVR training data into a multiple-choice task, where for each instance, three labels of the same category are randomly sampled as distractors.
2. As mentioned in [our paper](https://arxiv.org/pdf/2503.12797), since the base model (Qwen2-VL-7B-Instruct) already demonstrates strong FGVR capabilities, we trained it on the FGVR dataset using only GRPO. Therefore, there was no need to generate CoT training data — the model’s reasoning ability on FGVR comes entirely from GRPO training.
3. Following DeepSeek-R1, we adopted a rule-based reward function. During FGVR training, the reward consists of an accuracy reward and a format reward: the former is based on whether the model’s answer is correct, while the latter is based on whether the model’s output adheres to the required format. The training code will be released soon.

**Comment by StevenSmith2000** [comment]
Thanks for your answer!
